packages/server/dataloader/pollsLoaders.ts

```diff
@@ -0,0 +1,46 @@
+import DataLoader from 'dataloader'
+import RethinkDataLoader from './RethinkDataLoader'
+import getPollOptionsByPollId from '../postgres/queries/getPollOptionsByPollId'
+import getPollByDiscussionId from '../postgres/queries/getPollsByDiscussionId'
+import getPollsById from '../postgres/queries/getPollsById'
+import {IGetPollOptionsByPollIdQueryResult} from '../postgres/queries/generated/getPollsOptionsByPollIdQuery'
+import {IGetPollsByIdQueryResult} from '../postgres/queries/generated/getPollsByIdQuery'
+import {IGetPollsByDiscussionIdQueryResult} from '../postgres/queries/generated/getPollsByDiscussionIdQuery'
+
+export const pollOptions = (parent: RethinkDataLoader) => {
+  return new DataLoader<number, IGetPollOptionsByPollIdQueryResult[], string>(
+    async (pollIds) => {
+      const rows = (await getPollOptionsByPollId(pollIds)) || []
```

Comment:
I think if the query returns nothing for the `pollIds`, it will return an empty array so this feels redundant